Source/WebCore/css/CSSPrimitiveValueMappings.h

```diff
@@ -2410,6 +2411,38 @@ template<> constexpr FontOpticalSizing fromCSSValueID(CSSValueID valueID)
     return FontOpticalSizing::Enabled;
 }
 
+template<> constexpr FontTech fromCSSValueID(CSSValueID valueID)
```

Comment:
Can we not use abbreviations? We should spell out “technology”

Reply:
I agree, this isn't a web-exposed enum, so we can spell it out as it is.